### PR TITLE
Get Started with KKP on Single Node using Kubeone

### PR DESCRIPTION
**Pull request by archups**
- Documentation to demonstrate the KKP master deployment as kubeone addon on single node using kubeone.

Signed-off-by: archups <archupsawant@gmail.com>

### CODE REVIEW

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone"
+date = 2022-02-07T12:07:15+02:00
+weight = 10
+enableToc = true
++++
+
+# Get started with KKP - “KKP on single node with Kubeone”
+
+This chapther will guide you through the KKP Master setup on a single master/worker k8s node using Kubeone. Here, we are leveraging the kubeone addons capability for quick installation of KKP.
```
Comment (stroebitzer) [inline content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:11]: IMO chapther should be chapter.

Comment (archups) [reply content/kubermatic/master/installation/install_kkp_on_node/_index.en.md]: Thanks @stroebitzer. Fixed the typo error!

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
++++
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone"
+date = 2022-02-07T12:07:15+02:00
+weight = 10
+enableToc = true
++++
+
+# Get started with KKP - “KKP on single node with Kubeone”
```
[the remark this answers is not in the document]
Comment (archups) [reply content/kubermatic/master/installation/install_kkp_on_node/_index.en.md]: Removed the redundant title.

**Comment by archups** [comment]
> I think this should be a rather kubeone doc where a person could init a 1 node cluster, ragardless of KKP.
> 
> And that later we should link to that doc from KKP installation.

@kron4eg @scheeles @stroebitzer @xmudrii - Open to suggestion on where to place this guide. Can we finalize it, if we are good with content?

**Comment by scheeles** [comment]
@shaase-ctrl PTAL

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+* Get the LoadBalancer External IP by following command.
+  ```bash
+  kubectl get svc -n ingress-nginx
+  ```
+
+* Update DNS mapping with External IP of the nginx ingress controller service. Incase of AWS, the CNAME mapping for $TODO_DNS with External IP should be created for resolution.
+
+* Nginx Ingress Controller Load Balancer configuration - Add the node to backend pool manually.
+> Should be supported in future as part of Feature request[#1822](https://github.com/kubermatic/kubeone/issues/1822)
```
Comment (shaase-ctrl) [inline content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:80]: ... in the future

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
++++
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone”"
```
Comment (shaase-ctrl) [inline content/kubermatic/master/installation/install_kkp_on_node/_index.en.md:3]: Suggestion: Get started with KKP - KKP on a single node with KubeOne on AWS

We could copy this and make guides for digitalocean, azure etc. This kind of content would help a lot.

**Comment by shaase-ctrl** [comment]
> 

Yes and no. From my perspective this helps getting it on with KKP and KubeOne is just a tool to do that. Another entry should be single node kubeone clusters

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+> Should be supported in future as part of Feature request[#1822](https://github.com/kubermatic/kubeone/issues/1822)
+
+* Validate the Kubermatic resources and certificates.
+  ```bash
+  kubectl -n kubermatic get deployments,pods
+  ```
+  ```bash
+  kubectl get certificates -A -w
+  ```
+  > Wait for while, if still kubermatic-api-xxx pods / kubermatic & dex tls certificates are not in Ready state, delete  and wait to get validated.
```
[the remark this answers is not in the document]
Comment (archups) [reply content/kubermatic/master/installation/install_kkp_on_node/_index.en.md]: taken care!

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md (hunk 4 above)
Comment (archups) [reply content/kubermatic/master/installation/install_kkp_on_node/_index.en.md]: @shaase-ctrl  Updated the title as per Suggestion

File: content/kubermatic/master/installation/install_kkp_on_node/_index.en.md (hunk 3 above)
Comment (archups) [reply content/kubermatic/master/installation/install_kkp_on_node/_index.en.md]: Done!

**Comment by stroebitzer** [comment]
@shaase-ctrl where should we put this docu? KKP or KubeOne?

**Comment by shaase-ctrl** [comment]
@stroebitzer KKP Docu of course. But we can recycle parts within KubeOne as well - double tap.

**Comment by archups** [comment]
@shaase-ctrl can you please merge this PR as we have addressed comments? Can you help with moving https://github.com/kubermatic-labs/kkp-on-node repository under "kubermatic" organization. Thank you

**Comment by shaase-ctrl** [comment]
I think you need to open another PR to the correct repository @archups

**Comment by archups** [comment]
@shaase-ctrl  can you please assist with which repository we need to place this installation document?

**Pull request merged**